pieman.sh

```diff
@@ -323,15 +325,7 @@ case "${BUILD_TYPE}" in
 
     mount "${LOOP_DEV}p1" "${MOUNT_POINT}"
 
-    if [[ "${DEVICE}" == "opi-pc-plus" ]]; then
-        "${TOOLSET_FULL_PATH}/uboot-${UBOOT_VER}"/mkimage -C none -A arm -T script -d "${PIEMAN_DIR}"/files/opi/boot-pc-plus.cmd "${BOOT}"/boot.scr
-    fi
-
-    if [[ "${DEVICE}" == "opi-zero" ]]; then
-        "${TOOLSET_FULL_PATH}/uboot-${UBOOT_VER}"/mkimage -C none -A arm -T script -d "${PIEMAN_DIR}"/files/opi/boot-zero.cmd "${BOOT}"/boot.scr
-    fi
-
-    rsync -a "${BOOT}"/ "${MOUNT_POINT}"
+    rsync -a --no-o --no-g "${BOOT}"/ "${MOUNT_POINT}"
```

Comment:
It seems it's related to a different task. Why did you decide to add `--no-o` and `--no-g` here?

Reply:
Otherwise I get
```
rsync: chown "/home/denis/Documents/git/pieman/build/e79a5a1f-7d52-4d37-ace1-d7339b23e68a/mount_point/." failed: Operation not permitted (1)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1183) [sender=3.1.1]
```